### PR TITLE
ci(sentry): disable sentry in compose files and CI jobs

### PR DESCRIPTION
**Pull request by shivaji-dgraph**

### CODE REVIEW

**Review by mangalaman93 — DISMISSED** [review]

**Comment by MichelDiz** [comment]
qq, do we use contrib files in CI stuff?

**Comment by mangalaman93** [comment]
> qq, do we use contrib files in CI stuff?

probably not. Can you try removing it and see whether CI jobs fail in a separate PR?

**Comment by MichelDiz** [comment]
@mangalaman93 I think we don't. Cuz all composer files are contextual by path. It is rare point to external .yml - We should keep contrib as it is.

**Comment by joshua-goldstein** [comment]
> > qq, do we use contrib files in CI stuff?
> 
> probably not. Can you try removing it and see whether CI jobs fail in a separate PR?

We do use the Dockerfile there to build images, but other than that I don't believe anything from contrib is in CI.  I agree we don't need to change the docker-compose files there.  @shivaji-dgraph Could you make that quick change?

File: contrib/config/datadog/docker-compose.yml [hunk 1]
```diff
@@ -14,7 +14,7 @@ services:
       source: $GOPATH/bin
       target: /gobin
       read_only: true
-    command: /gobin/dgraph  ${COVERAGE_OUTPUT} alpha -o 100 --my=alpha1:7180 --zero=zero1:5080 --logtostderr -v=2
+    command: /gobin/dgraph  ${COVERAGE_OUTPUT} alpha --telemetry=sentry=false  -o 100 --my=alpha1:7180 --zero=zero1:5080 --logtostderr -v=2
```
[the remark this answers is not in the document]
Comment (shivaji-dgraph) [reply contrib/config/datadog/docker-compose.yml]: @MichelDiz Done.

**Review by mangalaman93 — APPROVED** [review]

**Review by joshua-goldstein — APPROVED** [review]

**Review by MichelDiz — APPROVED** [review]

**Pull request merged**